Bookish/Bookish.Web/Controllers/HomeController.cs

```diff
@@ -46,9 +46,9 @@ public IActionResult Book(string isbn)
             return View(new BookViewModel(book));
         }
 
-        public IActionResult AddBook()
+        public IActionResult AddBook(string isbn = "", string title = "", string authors = "", int copies = 1, bool isbnError = false)
```

Comment:
You shouldn't pass the isbn error as a url parameter. That way the user can change the url and get you to display a different error message. Instead I'd recommend doing the isbn check again here 🙂